### ISSUE TITLE
jquery UI postprocessing issue. Image 'animated-overlay.gif' not found

### ISSUE THREAD

**Issue by DheerendraRathor** [geex-arts/django-jet#18]
Hi, 
On production server, collectstatic is failing as it is unable to find 'animated-overlay.gif'

```bash
Post-processed 'jet/vendor/jquery-ui-timepicker/include/ui-1.10.0/ui-lightness/images/ui-icons_222222_256x240.png' as 'jet/vendor/jquery-ui-timepicker/include/ui-1.10.0/ui-lightness/images/ui-icons_222222_256x240.ebe6b6902a40.png'
Post-processing 'jet/vendor/jquery-ui-timepicker/include/ui-1.10.0/ui-lightness/jquery-ui-1.10.0.custom.min.css' failed!

Traceback (most recent call last):
  File "./manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/home/dheerendra/.Envs/ldapsso/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 350, in execute_from_command_line
    utility.execute()
  File "/home/dheerendra/.Envs/ldapsso/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 342, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/dheerendra/.Envs/ldapsso/local/lib/python2.7/site-packages/django/core/management/base.py", line 348, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/dheerendra/.Envs/ldapsso/local/lib/python2.7/site-packages/django/core/management/base.py", line 399, in execute
    output = self.handle(*args, **options)
  File "/home/dheerendra/.Envs/ldapsso/local/lib/python2.7/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 176, in handle
    collected = self.collect()
  File "/home/dheerendra/.Envs/ldapsso/local/lib/python2.7/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 128, in collect
    raise processed
ValueError: The file 'jet/vendor/jquery-ui-timepicker/include/ui-1.10.0/ui-lightness/images/animated-overlay.gif' could not be found with <django.contrib.staticfiles.storage.CachedStaticFilesStorage object at 0x7f8d62ed6a50>
```

**Comment by f1nality** [comment]
Thanks for your report! Fixed in 0.1.3.

**Comment by DheerendraRathor** [comment]
Thanks for fix. Working great now!
